packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/services/create_pin_code_service.dart

```diff
@@ -41,6 +41,7 @@ class CreatePinCodeService implements PinCodeService {
 
   @override
   Future<bool> verifyPinCode(String pinCode) async {
+    await Future.delayed(const Duration(seconds: 1));
```

Comment:
We need to include these only if the PIN animation is working. Otherwise it just appears like the app freezes.